pkg/globalnet/controllers/controllers_suite_test.go

```diff
@@ -61,7 +63,7 @@ const (
 	globalIngressIPName       = "nginx-ingress-ip"
 	kubeProxyIPTableChainName = "KUBE-SVC-Y7DIXXI5PNAUV7FB"
 	serviceName               = "nginx"
-	cniInterfaceIP            = "10.20.30.40"
+	cniInterfaceIP            = "169.254.1.50"
```

Comment:
Why do we need to update 'cniInterfaceIP' value ? 
 cniInterfaceIP shouldn't be from globalIP cidr range